### PR TITLE
OC-718 Add pinging to solve CDMA SYN/ACK issues

### PR DESCRIPTION
**Pull request by bvdzwet**

### CODE REVIEW

**Comment by bvdzwet** [comment]
retest this please

File: osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/CommandExecutor.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CommandExecutor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CommandExecutor.class);
```
Comment (harrymiddelburgetall) [inline osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/CommandExecutor.java:31]: I would preferred use of the @Slf4j annotation

File: osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/PingFlavor.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            ? "-n"
+                    : ""),
+    MAC_OS("ping", "-c %d", "-s %d", "-t %d", Duration::getSeconds, lookupNames -> Boolean.FALSE.equals(lookupNames)
+            ? "-n"
+                    : ""),
+    WINDOWS("ping", "-n %d", "-l %d", "-w %d", Duration::toMillis, lookupNames -> Boolean.TRUE.equals(lookupNames)
+            ? "-a"
+                    : "");
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PingFlavor.class);
```
Comment (harrymiddelburgetall) [inline osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/PingFlavor.java:32]: I would preferred use of the @Slf4j annotation

File: osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/Pinger.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Predicate;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Pinger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Pinger.class);
```
Comment (harrymiddelburgetall) [inline osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/Pinger.java:25]: I would preferred use of the @Slf4j annotation

**Review by harrymiddelburgetall — APPROVED** [review]
Tested on 4 devices with and without responses on ping. Positive result.
Codewise: I would preferred use of the @Slf4j annotation

**Pull request merged**